pdc_client/__init__.py

```diff
@@ -120,12 +121,25 @@ class to perform requests.
             except KeyError:
                 print("'%s' must be specified in configuration file." % CONFIG_URL_KEY_NAME)
                 sys.exit(1)
-            ssl_verify = config.get(CONFIG_SSL_VERIFY_KEY_NAME) if ssl_verify is None else ssl_verify
-            insecure = config.get(CONFIG_INSECURE_KEY_NAME)
-            if insecure is not None:
-                sys.stderr.write("Warning: '%s' option is deprecated; please use '%s' instead\n" % (
-                    CONFIG_INSECURE_KEY_NAME, CONFIG_SSL_VERIFY_KEY_NAME))
-                ssl_verify = not insecure
+            if ca_cert_or_insecure is None:
+                ssl_verify = config.get(CONFIG_SSL_VERIFY_KEY_NAME)
+                insecure = config.get(CONFIG_INSECURE_KEY_NAME)
+                if ssl_verify == insecure:
+                    # Give a warning if ssl_verify == insecure in config file
+                    print("In config file, the values of ssl_verify and insecure can't be the same")
+                    sys.exit(1)
+                if insecure is not None:
+                    sys.stderr.write("Warning: '%s' option is deprecated; please use '%s' instead\n" % (
+                        CONFIG_INSECURE_KEY_NAME, CONFIG_SSL_VERIFY_KEY_NAME))
+                    if insecure:
+                        ssl_verify = not insecure
+                    else:
+                        # ssl_verify can be None here.
```

Comment:
this comment is bit confused for me. which ssl_verify you mean can be None?
i guess you mean the second value.
But maybe you can say the value for the first ssl_verify can be a path or not insecure